html/manifest/share_target.json

```
@@ -0,0 +1,55 @@
+{
+  "html": {
+    "manifest": {
+      "share_target": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/share_target",
+          "spec_url": "https://w3c.github.io/manifest/#incubations",
```

Is this a better spec_url?

```suggestion
          "spec_url": "https://w3c.github.io/web-share-target/#share_target-member",
```

---

@tomayac Which spec should we list here?

---

https://w3c.github.io/web-share-target/#share_target-member is the better one indeed. The incubations spec actually points there, too.